packages/shared-components/src/components/onto-header/onto-header.tsx

```diff
@@ -1,18 +1,39 @@
 import { Component, Host, h } from '@stencil/core';
+import {WorkbenchServiceProvider, WorkbenchEventService, WorkbenchEventType} from "@ontotext/workbench-api";
 
 @Component({
   tag: 'onto-header',
   styleUrl: 'onto-header.scss',
   shadow: false,
 })
 export class OntoHeader {
+
+  private eventService: WorkbenchEventService;
+
+  private locale = 'en'
+
+  constructor() {
+    this.eventService = WorkbenchServiceProvider.get(WorkbenchEventService);
+  }
+
+  private onClick(): void {
```

Comment:
There is better way to name these handlers. The event which triggers the handler is `click` but the method does the language change logic. So it should be named changeLanguage or something.

Reply:
This is code just for testing. It will be removed in next request when language selector is implemented.

Reply:
Commented